### PR TITLE
Fix solver packages from graph-refresh job

### PR DESCRIPTION
**Pull request by fridex**
Fixes: https://github.com/thoth-station/solver/issues/12

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -114,11 +114,9 @@ def run_solver(solver: str, packages: str) -> str:
                 "env": [
                     {"name": "THOTH_SOLVER", "value": str(solver)},
                     # No need to run transitive again
-                    {"name": "THOTH_SOLVER_NO_TRANSITIVE", "value": "1"},
-                    {"name": "THOTH_SOLVER_PACKAGES", "value": str(
-                        packages.replace('\n', '\\n'))},
-                    # TODO: hardcoded now
-                    {"name": "THOTH_SOLVER_DEBUG", "value": "1"},
+                    {"name": "THOTH_SOLVER_NO_TRANSITIVE", "value": "0"},
```
Comment (goern) [inline app.py]: why no transitives?

Comment (fridex) [reply app.py]: This will turn on transitive solving.

Comment (goern) [reply app.py]: ja, why not?

Comment (fridex) [reply app.py]: If a transitive dependency introduces a dependency that was not previously analyzed, we do not have full stack in the graph database.

**Comment by fridex** [comment]
/approve

Removed the implicit value as discussed on the call yesterday.

**Pull request merged**